Sources/Sentry/SentryHub.m

```diff
@@ -32,12 +32,20 @@ - (instancetype)initWithClient:(SentryClient *_Nullable)client andScope:(SentryS
 - (void)startSession {
     SentrySession *lastSession = nil;
     SentryScope *scope = [self getScope];
+    SentryClient *client = [self getClient];
+    SentryOptions *options = [client options];
+    if (nil == options || nil == options.releaseName) {
+        [SentryLog logWithMessage:[NSString stringWithFormat:@"No option or release to start a session."] andLevel:kSentryLogLevelDebug];
+        return;
+    }
     @synchronized (_sessionLock) {
         if (nil != _session) {
             lastSession = _session;
         }
         _session = [[SentrySession alloc] init];
+        _session.releaseName = options.releaseName;
```

Comment:
Can we maybe add tests for this change?

Reply:
@philipphofmann what's your logaf scale here?

Reply:
start/end session still has no tests. Will come in a new PR (don't want to hold this PR until then)